recipes/textblob/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{%set name = "textblob" %}
+{%set version = "0.11.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "6fa856bd875ed8bb5dfb5e958f0ad4c3360362eb85b398370d98c3f244172052" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - nltk >=3.1
```

Comment:
Do you want to try adding `nltk`, @pmlandwehr? This would be a very nice addition. 😉